src/software/ai/hl/stp/tactic/receiver/receiver_tactic_test.cpp

```diff
@@ -191,10 +191,11 @@ INSTANTIATE_TEST_CASE_P(
         //                                  Angle::fromDegrees(0),
         //                                  Angle::fromDegrees(0))}),
 
-        std::make_tuple(Pass(Point(4.0, 1.5), Point(3.0, -1), 4.5),
-                        RobotStateWithId{1, RobotState(Point(3.0, -1), Vector(0, 0),
-                                                       Angle::fromDegrees(180),
-                                                       Angle::fromDegrees(0))}),
+        // TODO (#2570): re-enable when one-touch works for these tests
+        // std::make_tuple(Pass(Point(4.0, 1.5), Point(3.0, -1), 4.5),
+        //                 RobotStateWithId{1, RobotState(Point(3.0, -1), Vector(0, 0),
+        //                                                Angle::fromDegrees(180),
+        //                                                Angle::fromDegrees(0))}),
```

Comment:
Why is one-touch not working? Is it related to this PR?

Reply:
I think at some point this test was failing on this branch, but just tried it again and it seems to be passing. So I've re-enabled it